packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/e2e/fixture.ts

```diff
@@ -0,0 +1,42 @@
+import {RecentResultListObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {searchRequestRegex} from '../../../../../../playwright/utils/requests';
+import {AnalyticsModeEnum} from '../../../../../../playwright/utils/analyticsMode';
+
+const pageUrl = 's/quantic-recent-results-list';
+
+interface RecentResultListOptions {
+  engineId: string;
+}
+
+type QuanticRecentResultListE2ESearchFixtures = {
```

Comment:
```suggestion
type QuanticRecentResultsListE2ESearchFixtures = {
```

I think you're missing an S everywhere.